### ISSUE TITLE
Account softban

### ISSUE THREAD

**Issue by majinkrys6** [PokeBotPub/PokeBot-Android#345]
Hello my account is softban every 2 seconds when the softban is bypassed its start again ... what to do?

**Comment by Sangputri2006** [comment]
Can,t connect niantic server, what is problem pokebot v1. 0.045

**Comment by AbiDez** [comment]
The beginning of the end oO
I think that's funny days in next time :(

**Comment by Sangputri2006** [comment]
Now it is not running again, what happens

**Comment by majinkrys6** [comment]
Yes the maj 1.45 does and I am not banned I can connect and play normal

**Comment by majinkrys6** [comment]
Sorry I closed the post not done express

**Comment by ArthurPetrilli** [comment]
Don't log in with that acct for a week or two.  When you log in, play normal pogo.  Then use bot again.  Your acct has been flagged.  This happened to one of my test accts.  Since 1.0.40, i feel 1.0.45 is stable enough to run for about 60 minute increments without ill effects.  Too many softbans (level 27 and up) will most likely (90%) lead to permaban.

**Comment by ArthurPetrilli** [comment]
I have another acct (level 27) stopped working...can't connect to server.  After break, i attempted to login to pogo.  My android devices made me login....as if i was never logged in.....why is that????  I was able to login to pogo.  After another break, i had attempted to login to Pokebot.  I still get msg that i can't connect to server.  I only ran the bot 3x with each session lasting 60 minutes with 30-60 breaks between.  Only gained 50,000 xp on that acct for the day.  This is ridiculous if i can only log in 2x with each session at 60 minutes.  Another acct going into cold storage until a solution is found.....who know....this may be another acct on the permaban chopping block!  Why would using the bot on my device make it forget or act like suspicious activity was going on????  Hmmm, do i continue going with other accts knowing that i will get the same results.....sounds like insanity.  Maybe there will be a Pokebot 1.0.46 or it is bye bye level 27.
